en/components/dialog.md

```diff
@@ -4,87 +4,180 @@ _description: Whether building informative dialogs or data manipulation windows,
+</div>
+<div class="divider--half"></div>
 
-`leftButtonBackgroundColor`
+####Custom Dialog
 
-`leftButtonRipple`
-<div class="divider--half"></div>
+To add custom dialog, in the template of our sign in component we can add the following code to get the custom dialog. We have to set the `title`,`leftButtonLabel`,`rightButtonBackgroundColor`, `rightButtonColor`, `rightButtonLabel`, `closeOnOutsideSelect` and handle `onLeftButtonSelect` and `onRightButtonSelect` event.
+Also we can add two groups of label and input decorated with the [**igxLabel**](../components/label_input.html) and [**igxInput**](../components/label_input.html) directives.
 
-###Custom Dialogs
-**Custom Dialogs are done by adding any markup in the igx-dialog tag.**
-When you are using Custom Dialogs you don't have a message property set.
+```html
+<!--sign-in.component.html-->
 
-```HTML
 <igx-dialog #form
     title="Sign In"
     leftButtonLabel="Cancel"
     (onLeftButtonSelect)="form.close()"
+    (onRightButtonSelect)="signIn($event)"
     rightButtonLabel="Sign In"
-    rightButtonBackgroundColor="green"
-    rightButtonColor="white"
+    rightButtonBackgroundColor="#0375be"
+    rightButtonColor="#ffffff"
     closeOnOutsideSelect="true">
     <div class="ig-form-group">
-        <input type="text" igInput />
-        <label igLabel>Username</label>
+        <input type="text" igxInput />
+        <label igxLabel>Username</label>
     </div>
     <div class="ig-form-group">
-        <input type="password" igInput />
-        <label igLabel>Password</label>
+        <input type="password" igxInput />
+        <label igxLabel>Password</label>
     </div>
 </igx-dialog>
 ```
 
-You can make the dialog dismissible `closeOnOutsideSelect="true"``
+<div class="sample-container loading" style="height:300px">
+    <iframe id="dialog-sample-3-iframe" src='{environment:demosBaseUrl}/dialog-sample-3' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn" data-iframe-id="dialog-sample-3-iframe" data-demos-base-url="{environment:demosBaseUrl}">view on stackblitz</button>
+</div>
+<div class="divider--half"></div>
+
+### API Summary
+The following tables summarize some of the useful **igx-dialog** component inputs, outputs and methods.
+
+#### Inputs
+The following inputs are available in the **igx-dialog** component:
+
+| Name | Type | Description |
+| :--- | :--- | :--- |
+| `id` | string | Unique identifier of the dialog. |
+| `title` | string | Specifies the title of the dialog. |
+| `message` | string | Specifies the message of the dialog. |
+| `leftButtonLabel` | string | When set, the left button becomes visible. |
+| `leftButtonType` | string | Specifies the left button [**type**](button.md#button-types). Default value is flat. |
```

Comment:
I'm pretty sure you can't link to .md files in production.

Reply:
Basically there is a parser, which converts the .md files to html files, so this should work on production

Reply:
We are going to review the changes after merging them to staging.